roles/nextflow/templates/nextflow_miarka_site.config.j2

```diff
@@ -1,5 +1,9 @@
 
-cleanup = true
+{% if site == "upps" %}
+    cleanup = false
+{% else %}
+    cleanup = true
```

Comment:
This is a nitpick, but I don't think this line should be indented. It probably works like this, but to keep it tidy and easier to reed I think it should be on "root level".

Reply:
Agreed, updated accordingly!